tests/tests.js

```diff
@@ -542,18 +542,16 @@ describe('unicodePropertyEscapes', () => {
 			const p = `\\p{${ pattern }}`;
 			it('rewrites `/' + p + '/u` correctly', () => {
 				const transpiled = rewritePattern(p, 'u', features);
-				assert(
-					transpiled == expected.p ||
-					transpiled == '(?:' + expected.p + ')'
-				);
+				if (transpiled != '(?:' + expected.p + ')') {
+					assert.equal(transpiled, expected.p);
+				}
```

Comment:
This is not really related, but I had to debug an error and the error message was completely unhelpful (`expected true; actual false`).

Reply:
Ack. Open to better suggestions!